server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -423,20 +424,28 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 	public handler(rawMessage: IQueuedMessage) {
 		// In cases where we are reprocessing messages we have already checkpointed exit early
 		if (rawMessage.offset <= this.logOffset) {
-			Lumberjack.info(
-				`rawMessage.offset: ${rawMessage.offset} <= this.logOffset: ${this.logOffset}`,
-				getLumberBaseProperties(this.documentId, this.tenantId),
-			);
+			const reprocessOpsMetric = Lumberjack.newLumberMetric(LumberEventName.ReprocessOps);
+			reprocessOpsMetric.setProperties({
+				...getLumberBaseProperties(this.documentId, this.tenantId),
+				kafkaMessageOffset: rawMessage.offset,
+				databaseLastOffset: this.logOffset,
+			});
 
 			this.updateCheckpointMessages(rawMessage);
 
-			if (this.checkpointInfo.currentKafkaCheckpointMessage) {
+			if (
+				this.checkpointInfo.currentKafkaCheckpointMessage &&
+				this.kafkaCheckpointOnRestart
+			) {
 				this.context.checkpoint(
 					this.checkpointInfo.currentKafkaCheckpointMessage,
 					this.restartOnCheckpointFailure,
 				);
+				reprocessOpsMetric.setProperty("kafkaCheckpointOnRestart", true);
 			}
-
+			reprocessOpsMetric.success(
+				`Repeating ops: rawMessage.offset: ${rawMessage.offset} <= this.logOffset: ${this.logOffset}`,
```

Comment:
Instead of put these two values in message, could we set these two messages into `properties`?

Reply:
They are included in the properties, but I can update the message to improve our logs